content/es-AR/learn/getting-started.smd

```diff
@@ -98,7 +98,7 @@ scoop install versions/zig-dev
 
 ### macOS
 
-**Homebrew**  
```

Comment:
careful, double spaces at the end of a line have semantic meaning in markdown

Reply:
Thank you, reviewing.